dds.sh

```diff
@@ -1,6 +1,6 @@
 package: DDS
 version: "3.5.1"
```

Comment:
Could you change also the version, or use tag_basename?

Reply:
Done.